docs/release-notes/index.md

```diff
@@ -70,6 +70,7 @@ v7
     The next Codacy Self-hosted version should be a major version
     See the breaking changes introduced in https://codacy.atlassian.net/browse/CY-5848-->
```

Comment:
We decided to release this version of Codacy Self-hosted as a new major version.

I'm re-using this pull request to change the release notes to version 8.0.0 instead of 7.1.0.